Dockerfile

```diff
@@ -1,9 +1,12 @@
 FROM golang:buster
-ARG GOPROXY=https://goproxy.cn,direct
 
-COPY ./src /app
+COPY ./src/go.mod /app/
+COPY ./src/go.sum /app/
 WORKDIR /app
+
 RUN go mod download
```

Comment:
With these changes, Docker won't download requirements every time we change the code